packages/web/src/components/layouts/Website/Website.tsx

```diff
@@ -19,39 +18,38 @@ export type Props = {
   //   title: string;
   //   icon: string;
   // }>;
-  packages: DetectedPackageData[];
+  isLoading: boolean;
+  isPending: boolean;
+  packages: WebPagePackage[];
   vulnerabilities: Record<string, PackageVulnerabilityData[]>;
-  webpages: Array<{
-    status: string;
-  }>;
-  onFiltersApply: SubmitHandler<FormData>;
+  webpages: WebPage[];
```

Comment:
I believe we should strictly use `trpc` types instead of database entities on the client, since some fields data types might be changed (`Date` -> `string` for example)